src/formatDate.js

```diff
@@ -50,7 +50,51 @@
  */
 
 function formatDate(date, fromFormat, toFormat) {
-  // write code here
+  const from = fromFormat[3];
+  const to = toFormat[3];
+
+  const obj = {};
+  const result = [];
+  const value = date.split(from);
+
+  for (let i = 0; i < 3; i++) {
```

Comment:
```suggestion
  for (let i = 0; i < fromFormat.length; i++) {
```
if you iterating through the array, use array.length instead of 'magic numbers'